src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(str) {
+  const result = {};
```

Comment:
The variable name 'result' is too generic. Consider a more descriptive name like 'styleObject' to improve code readability.